datalad/distribution/install.py

```diff
@@ -322,21 +322,48 @@ def __call__(dataset=None, path=None, source=None, recursive=False,
 
         # resolve the target location against the provided dataset
         if path is not None:
-            path = resolve_path(path, ds)
+            # make sure it is not a URL, `resolve_path` cannot handle that
+            if is_url(path):
+                try:
+                    path = get_local_path_from_url(path)
+                    path = resolve_path(path, ds)
+                except ValueError:
+                    # URL doesn't point to a local something
+                    pass
+            else:
```

Comment:
codecov extension for chromium finally started to work and it is awesome, e.g. for above code note lines marked red ;) :  ![above snipped ](http://www.onerussian.com/tmp/gkrellShoot_05-21-16_222643.png)